### PR TITLE
Typo Update README.md

### PR DESCRIPTION
**Pull request by famouswizard**
### Description

This pull request addresses several minor typographical errors in the documentation, which collectively improve readability and clarity. The following changes have been made:

1. **Section "🔧 Configuration":**
   - **Typo**: "To getting started with Bandada, you can get rid of the configuration and just use the default settings."
   - **Correction**: "To get started with Bandada, you can skip the configuration and just use the default settings."
   - **Importance**: This change clarifies the intended meaning, making it easier for new users to understand setup instructions.

2. **Section "🔌 API endpoints":**
   - **Typo**: "APIs endpoints"
   - **Correction**: "API endpoints"
   - **Importance**: The corrected phrasing follows conventional terminology, which improves the technical accuracy and consistency of the document.

3. **Section "👨‍💻 Contributing - Code quality and formatting":**
   - **Typo**: "Run Prettier to check formatting rules."
   - **Correction**: "Use Prettier to check formatting rules."
   - **Importance**: This rephrasing makes it clear that Prettier is a tool to use, not an action to run, which better reflects its role in the workflow.

4. **Section "👨‍💻 Contributing - Conventional commits":**
   - **Typo**: "Bandada utilises conventional commits..."
   - **Correction**: "Bandada uses conventional commits..."
   - **Importance**: This change standardizes spelling for accessibility and broadens the documentation's usability for international readers.

5. **Section "👨‍💻 Contributing - Releases":**
   - **Typo**: "After pushing the new git tag, a workflow will be triggered and will publish the Bandada packages on npm and release a new version on Github with its changelogs automatically."
   - **Correction**: "After pushing the new git tag, a workflow will be triggered to publish the Bandada packages on npm and release a new version on GitHub with its changelogs automatically."
   - **Importance**: This revision clarifies the purpose and function of the automated workflow, reducing potential confusion for contributors managing releases.

**Impact:**  
These corrections make the documentation more professional, clearer, and easier for users and contributors to follow. By improving these details, the documentation aligns better with Bandada’s commitment to quality and developer-friendly resources.

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ All the Bandada board tasks are public. You can view the current tasks, their st
 
 -   [NodeJS](https://nodejs.org/en) >= v18.17.0
 
-Each package and application brings its own set of environment variables. To getting started with Bandada, you can get rid of the configuration and just use the default settings. Copy the environment variables used by the [`api`](/apps/api/) application by running this command:
```
Comment (vplasencia) [inline README.md:181]: I think it's better to keep using `to get started` here.

File: README.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ Run [ESLint](https://eslint.org/) to analyze the code and catch bugs:
 yarn lint
 ```
 
-Run [Prettier](https://prettier.io/) to check formatting rules:
```
Comment (vplasencia) [inline README.md:317]: I think `Run` is fine here since people need to run Prettier to check formatting.

**Review by vplasencia — CHANGES_REQUESTED** [review]
Hey @famouswizard thank you very much for this PR. I just left some comments.

File: README.md (hunk 2 above)
Comment (famouswizard) [reply README.md:317]: Perfect

Comment (vplasencia) [reply README.md:317]: Could you update it and put `Run` instead of `Use`?

**Review by vplasencia — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (famouswizard) [reply README.md:317]: Updated.

Comment (vplasencia) [reply README.md:317]: Hey! Thank you! I think that the update is not reflected here. I think you need to update the branch you used to create the PR (`patch-1`) and the changes will be reflected here automatically.

**Review by vplasencia — APPROVED** [review]

**Pull request merged**